x-pack/plugins/security_solution/public/cases/containers/use_get_case.tsx

```diff
@@ -121,17 +122,19 @@ export const useGetCase = (caseId: string): UseGetCase => {
         }
       }
     };
+    didCancel.current = false;
+    abortCtrl.current.abort();
+    abortCtrl.current = new AbortController();
     fetchData();
-    return () => {
-      didCancel = true;
-      abortCtrl.abort();
-    };
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [caseId]);
+  }, [caseId, subCaseId, dispatch, dispatchToaster]);
```

Comment:
I think is best to remove `dispatch` and `dispatchToaster`. They can create unnecessary re-renders